Assets/Scripts/Player.cs

```diff
@@ -49,9 +51,60 @@ public class Player : MonoBehaviour
 
 	private string openVideo;
 
-	void Start ()
+	private static class ConnectedDevices
 	{
-		XRSettings.enabled = false;
+		public enum LoadedDevice
```

Comment:
Notes:

I would arrange the LoadedDevice enum in such a way that you can use it for gracious fallback. i.e. Oculus is more specific than OpenVr, so you could say `if (loadedDevice > LoadedDevice.OpenVr)`.